project/Settings.scala

```diff
@@ -5,7 +5,7 @@ import sbtassembly.AssemblyKeys._
 object Settings {
   val buildName = "hazelcast-spark"
   val buildVersion = "0.1-SNAPSHOT"
-  val buildScalaVersion = "2.10.5"
+  val buildScalaVersion = "2.11.8"
```

Comment:
@eminn maybe there is some sense to update Scala to 2.10.6 It's last version in 2.10.*